engine/src/test/scala/cromwell/ArrayAndMapIndexingWorkflowSpec.scala

```diff
@@ -10,10 +10,10 @@ import scala.language.postfixOps
 
 class ArrayAndMapIndexingWorkflowSpec extends CromwellTestkitSpec {
   "A workflow with array/map indexes in expressions" should {
-    "run locally" ignore {
+    "run locally" in {
       runWdlAndAssertOutputs(
         sampleWdl = SampleWdl.ArrayAndMapIndexingWdl,
-        eventFilter = EventFilter.info(pattern = s"starting calls: test.echo_int, test.echo_str", occurrences = 1),
+        eventFilter = EventFilter.info(pattern = "Workflow complete", occurrences = 1),
```

Comment:
previously this was checking some, but not all of the calls starting (generally) in all the workflow tests.  I don't think it was adding much value so now it is just checking for the log message of Workflow complete.  As we invest more in the logging of the call actors, etc we can beef these up if we want